gnomad/utils/annotations.py

```diff
@@ -1106,7 +1108,8 @@ def region_flag_expr(
     :return: `region_flag` struct row annotation
     """
     prob_flags_expr = (
-        {"non_par": (t.locus.in_x_nonpar() | t.locus.in_y_nonpar())} if non_par else {}
+        {"non_par": (t.locus.in_x_nonpar() | t.locus.in_y_nonpar())
+         } if non_par else {}  # fmt: skip
```

Comment:
Ha the extra note made the line long enough that black wants parentheses there now...

Reply:
oh boy. I'm surprised it was even removing the parentheses, mine doesn't do that